PROVESFlightControllerReference/test/yamcs/conftest.py

```
@@ -0,0 +1,56 @@
+    deadline = time.time() + YAMCS_READY_TIMEOUT_S
+    last_error: Exception | None = None
+    while time.time() < deadline:
+        try:
+            instance = next(
+                (i for i in client.list_instances() if i.name == YAMCS_INSTANCE),
+                None,
+            )
+            if instance is not None and getattr(instance, "state", None) == "RUNNING":
+                yield client
+                return
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)